README.md

```diff
@@ -38,11 +38,21 @@ You can run the model [here](https://colab.research.google.com/github/oughtinc/e
 
 ## Quickstart
 
+### In your browswer
 1. Open [this Colab](https://colab.research.google.com/github/oughtinc/ergo/blob/master/notebooks/build/quickstart.ipynb)
 2. Add your Metaculus username and password
 3. Select "Runtime > Run all" in the menu
 4. Edit the code to load other questions, improve the model, etc., and rerun
 
+
+### On your computer
+1. Fetch and open the repo: `git clone git@github.com:oughtinc/ergo.git; cd ergo`
+2. If you don't have it installed already, [install Poetry](https://python-poetry.org/docs/#installation). Ergo uses this to manage its dependencies and environments.
+3. Use poetry to setup a virtual environment, and download dependencies: `poetry install`
```

Comment:
For what it's worth I think I've run into trouble at this step with `poetry` projects in the past because my Python environment [looked like this](https://xkcd.com/1987/) and I was accidentally trying to use some weird `miniconda` install.

So I'm not sure everyone will run into problems for the same reason, but still seems helpful to list out some common reasons for problems